libs/components/src/form-field/bit-validators/trim.validator.ts

```diff
@@ -0,0 +1,22 @@
+import { AbstractControl, FormControl, ValidationErrors, ValidatorFn } from "@angular/forms";
+
+export function trimValidator(): ValidatorFn {
+  return (control: AbstractControl): ValidationErrors | null => {
+    if (!(control instanceof FormControl)) {
+      throw new Error("trimValidator only supports validating FormControls");
+    }
+
+    if (control.value === null || control.value === undefined) {
+      return null;
+    }
+    const value = control.value;
+    if (value != value.trim()) {
+      return {
+        trim: {
+          message: "input is not trimmed",
+        },
+      };
+    }
+    return null;
+  };
+}
```

Comment:
We don't need an argument, so we should be able to rewrite this to something like:

```suggestion
export function trimValidator(control: AbstractControl): ValidationErrors | null => {
  if (!(control instanceof FormControl)) {
    throw new Error("trimValidator only supports validating FormControls");
  }

  if (control.value === null || control.value === undefined) {
    return null;
  }
  const value = control.value;
  if (value != value.trim()) {
    return {
      trim: {
        message: "input is not trimmed",
      },
    };
  }
  return null;
};
```

Which lets us add the validator with just `BitValidators.trimValidator`

Reply:
Resolved in https://github.com/bitwarden/clients/pull/4993/commits/3a13bf23eadd4f45c400c37002e8486fa8a0986d